### PR TITLE
New docs

### PR DESCRIPTION
**Pull request by johnBartos**
- Create CONTRIBUTING.md
- Update README.md

Style inspired by [Hoodie's Awesome Docs](https://github.com/hoodiehq/hoodie/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+````
+# Clone your fork of the repo into the current directory
+git clone https://github.com/<your-username>/jwplayer
+# Navigate to the newly cloned directory
+cd jwplayer
+# Assign the original repo to a remote called "upstream"
+git remote add upstream https://github.com/jwplayer/jwplayer
+````
+
+2. Install the dependencies:
```
Comment (egreaves) [inline BUILDING.md]: In the markup, the numbering isn't incremented for each item.

Comment (robwalch) [inline BUILDING.md]: @johnBartos removing the line breaks should fix that. Line numbers reset to 1 in md after a line break.

File: README.md [hunk 2]
```diff
@@ -3,64 +3,102 @@
...
+ - [Developer Portal](https://developer.jwplayer.com/)
+ - [API Reference](https://developer.jwplayer.com/jw-player/docs/developer-guide/api/javascript_api_reference/) 
+ - [Configuration Reference](https://developer.jwplayer.com/jw-player/docs/developer-guide/customization/configuration-reference/)
+- [Demos](https://developer.jwplayer.com/jw-player/demos/)
+- [Support](http://support.jwplayer.com/)
+
+
+## A Simple Example
+
+The example below and render a video player into the div with the `player` id, listens to an event, and makes a few calls using the API.
```
Comment (robwalch) [inline README.md]: > The example below and render a video player...

I don't quite understand this sentence.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+## Pull Requests
+Pull requests are a great way to help out and improve JW Player. Before embarking on a large PR, please ask us first - we may not need your changes in the repo and don't want you to waste your energy.
+
+To get your repo up and running, please refer to BUILDING.md.
+
+#### First Timers
+If you've never contributed before, don't worry - we'll do the best we can to help you throughout the process. Unsure of where to begin? Check out our [beginner bugs](https://github.com/jwplayer/jwplayer/labels/beginner), or refer to the Questions/Discussions section and drop us a line. 
+
+#### General Guidelines
+While far from comprehensive, following these guidelines will get the easy stuff out of the way and speed the process along. Be sure to include a good
```
Comment (robwalch) [inline CONTRIBUTING.md]: > Be sure to include a good

a good what?

**Pull request merged**